Tasks/NuGetAuthenticateV1/make.json

```diff
@@ -0,0 +1,22 @@
+{
+    "rm": [       
+        {
+            "items": [
+                "node_modules/azure-pipelines-tasks-artifacts-common/node_modules/azure-pipelines-task-lib"
+            ],
+            "options": "-Rf"
+        }
+    ],
+    "externals": {
+        "archivePackages": [
+            {
+                "url": "https://vstsagenttools.blob.core.windows.net/tools/NuGetCredProvider/1.0.0/c.zip",
+                "dest": "./ArtifactsCredProvider/"
+            },
+            {
+                "url": "https://vstsagenttools.blob.core.windows.net/tools/NuGetCredProvider/1.0.0/n6.zip",
+                "dest": "./ArtifactsCredProviderNet6/"
+            }
```

Comment:
These are the changed parts of V1 vs V0. Net 6 cred provider is a separate release asset for now so it's downloaded form a separate zip.